features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -168,5 +208,10 @@ class Base64Serializer private constructor(
     internal companion object {
         @VisibleForTesting
         internal const val BITMAP_SIZE_LIMIT_BYTES = 15000 // 15 kbs
+
+        internal const val DOES_NOT_IMPLEMENT_COMPONENTCALLBACKS =
+            "LruCache does not implement ComponentCallbacks2"
```

Comment:
The type is just `Cache`, so no necessary it is `LRU`.

```suggestion
            "Cache instance does not implement ComponentCallbacks2"
```